AsyncDisplayKit/ASTableView.mm

```diff
@@ -820,7 +820,9 @@ - (ASCellNode *)dataController:(ASDataController *)dataController nodeAtIndexPat
 {
   ASCellNode *node = [_asyncDataSource tableView:self nodeForRowAtIndexPath:indexPath];
   ASDisplayNodeAssert([node isKindOfClass:ASCellNode.class], @"invalid node class, expected ASCellNode");
-  node.layoutDelegate = self;
+  if (node.layoutDelegate == nil) {
```

Comment:
I want to allow developers to set their own layout delegate (in -tableView:nodeForRowAtIndexPath). This checks make sure if developer indeed provides such delegate, it is not overridden here.